src/rgw/rgw_process.cc

```diff
@@ -309,6 +324,16 @@ int process_request(rgw::sal::Store* const store,
 
 done:
   if (op) {
+    s->trace->SetTag(tracing::RETURN, op->get_ret());
+    if(s->user) {
```

Comment:
small nit: can you fix at instances: 
```suggestion
    if (s->user) {
```